src/lib/eliom_client.client.ml

```diff
@@ -544,22 +544,29 @@ module Page_status = struct
   end
 
   let maybe_just_once ~once e = if once then React.E.once e else e
+  let stop_event ?(stop = React.E.never) e =
+    ignore @@ React.E.map (fun () -> React.E.stop e) stop
```

Comment:
You need to use `React.E.stop ~strong:true`.
See http://erratique.ch/software/react/doc/React.html#strongstop